parts/linux/cloud-init/artifacts/cgroup-stats.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Updates certificates copied from AKS DS
+
+[Service]
+Type=oneshot
+ExecStart=/bin/bash /opt/scripts/cgroup-stats.sh
+RestartSec=5
```

Comment:
default `Restart=no` I believe, so this is actually not doing anything.

you can do `Restart=on-failure`, or just drop `RestartSec` and use the timer to hit the next interval. I'd expect this to be rare anyway, so maybe prefer that

Reply:
I'll just drop the RestartSec